### DISCUSSION TITLE
Great circle navigation

### DISCUSSION THREAD

**Discussion by schauveau** [Stellarium/stellarium#3972] [Q&A]
Is there a way to change the observer location by following great circle lines, if possible using the keyboard? 
 
I already know how to move using the Location window but the displacements can only follow a longitude or a latitude line. 

Ideally, I would like to be able to move forward, backward, left or right by a significant distance (e.g. 1° or 111km on Earth). 

When moving forward and backward the stars in front of the observer shall go up and down while the stars on both sides shall rotate.
Moving left or right shall do the opposite.

**Comment by gzotti** [comment]
There is no such function currently.

**Comment by schauveau** [comment]
@gzotti Thanks.

I just found the «Random location» script. I shall be able to modify it to produce the required displacements. 
Is there a way to start scripts without seeing the Configuration window or the Script Console window?

**Reply by gzotti** [reply]
User Guide, ch.4.3.6 or even 4.8.

**Comment by schauveau** [comment]
Thanks. I previously looked in the keyboard shortcut settings but I completely missed the fact that each script has an entry (added automatically I suppose) because when I searched for "script", the only match was for the Startup Script.
